extensions/git/src/main.ts

```diff
@@ -36,7 +36,9 @@ async function createModel(context: ExtensionContext, outputChannel: OutputChann
 	const pathHint = workspace.getConfiguration('git').get<string>('path');
 	const info = await findGit(pathHint, path => outputChannel.appendLine(localize('looking', "Looking for git in: {0}", path)));
 
-	let env: any = {};
+	let env: any = {
+		GIT_PAGER: ''
+	};
```

Comment:
Let's just do this here instead: https://github.com/microsoft/vscode/blob/95f26d2005ea0633aa48832d11e1f37068c4dcdb/extensions/git/src/git.ts#L541